debian/update-dch-from-git

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I think this functionality would just be clutter in the top-level Makefile - it's only of interest to the debian packaging, so it belongs in the debian/ directory.

I could see it maybe going in debian/rules, but I chose to put it in a separate file because I don't like maintaining multi-line shell scripts inside Makefiles.